home/ivp-planning/advanced-tools/path-tool.md

```diff
@@ -1,102 +1,80 @@
 # Path Tool
 
-The path tool is an easy but very powerful function of iVP that can be used for quick measurements as well as advanced highlighting. In its more specific forms - the[ fence tool ](fence-tool.md)respective [room tool ](the-room-tool.md)- it can also be used for the creation of complex [fences ](fence-tool.md#creating-a-fence)and [rooms ](the-room-tool.md#creating-a-room)in minutes.
+The path tool is an easy yet remarkably powerful function of iVP that can be utilized for quick measurements as well as advanced highlighting. In its various forms - including the [fence tool](fence-tool.md) and the [room tool](the-room-tool.md) - it can also expedite the creation of complex [fences](fence-tool.md#creating-a-fence) and [rooms](the-room-tool.md#creating-a-room) within minutes.
```

Comment:
```suggestion
The path tool is an easy yet remarkably powerful feature of Planning that can be utilized for quick measurements as well as advanced highlighting. In its various forms - including the [fence tool](fence-tool.md) and the [room tool](the-room-tool.md) - it can also expedite the creation of complex [fences](fence-tool.md#creating-a-fence) and [rooms](the-room-tool.md#creating-a-room) within minutes.
```

Additional to the suggestions above:
* Should mention the ground marking tool as well
* fences and rooms are mentioned twice in the last sentence, which sounds super repetetive